Source/Core/Core/PowerPC/PPCAnalyst.cpp

```diff
@@ -309,11 +311,15 @@ static void FindFunctionsAfterBLR(PPCSymbolDB* func_db)
   {
     while (true)
     {
-      // skip zeroes that sometimes pad function to 16 byte boundary (e.g. Donkey Kong Country
-      // Returns)
-      while (PowerPC::HostRead_Instruction(location) == 0 && ((location & 0xf) != 0))
+      // Skip zeroes (e.g. Donkey Kong Country Returns) and nop (e.g. libogc)
+      // that sometimes pad function to 16 byte boundary.
+      for (PowerPC::TryReadInstResult read_result = PowerPC::TryReadInstruction(location);
+           read_result.valid && (read_result.hex == 0 || read_result.hex == 0x60000000) &&
```

Comment:
All of these conditionals should probably be broken up into named identifiers or lambdas, this is getting kind of ridiculous. The body of the for loop should also be braced as well, since the loop header spans more than one line.